WFInfo/Ocr.cs

```diff
@@ -812,10 +812,10 @@ internal static void ProcessSnapIt(Bitmap snapItImage, Bitmap fullShot, Point sn
             snapItImageFiltered.Dispose();
             if (!File.Exists(applicationDirectory + @"\export " + DateTime.UtcNow.ToString("yyyy-MM-dd", Main.culture) + ".csv") && Settings.SnapitExport)
                 csv += "ItemName,Plat,Ducats,Volume,Vaulted,Owned," + DateTime.UtcNow.ToString("yyyy-MM-dd", Main.culture) + Environment.NewLine;
-
             foreach (var part in foundParts)
             {
-                if (part.Name.Length < 13) // if part name is smaller than "Bo prime handle" skip current part
+                if (part.Name.Length < 13 && Settings.locale == "en" || part.Name.Length < 2 && Settings.locale == "ko") // if part name is smaller than "Bo prime handle" skip current part 
```

Comment:
Add a minimum Korean prime part word length. Currently I've gone for two as I do not know the minimum. As close to the actual minimum is preferred.